Azure Monitor docs

daniellee opened the pull request:
Still have to add images and do some tidying up.

Fixes #15241

daniellee committed:
docs: adds Azure Monitor docs

A copy of the docs from the plugin with some additions and
new images.

marefr committed:
docs: fix order of datasources in menu/index and update alert support

marefr committed:
docs: fix link

marefr commented on docs/sources/features/datasources/azuremonitor.md:
```diff
@@ -0,0 +1,271 @@
+
+### Azure Monitor Metrics Whitelist
+
+Not all metrics returned by the Azure Monitor API have values. The Grafana datasource has a whitelist to only return metric names if it is possible they might have values. This whitelist is updated regularly as new services and metrics are added to the Azure cloud. You can find the current whitelist [here](https://github.com/grafana/grafana/public/app/plugins/datasource/grafana-azure-monitor-data ...
```
Broken link

marefr commented on docs/sources/features/datasources/azuremonitor.md:
```diff
@@ -0,0 +1,271 @@
+- Chaining template variables: `ResourceNames($rg, $ns)`
+- Do not quote parameters: `MetricNames(hg, Microsoft.Network/publicIPAddresses, grafanaIP)`
+
+{{< docs-imagebox img="/img/docs/v60/azuremonitor-service-variables.png" class="docs-image--no-shadow" caption="Nested Azure Monitor Template Variables" >}}
```
Broken image, but I guess you already know that

daniellee replied:
Not for me - did you download the latest from grafana.org and build the docker container?

daniellee replied:
Double checked that they are pushed and they were: https://github.com/grafana/grafana.org/tree/master/static/img/docs/v60

marefr commented on docs/sources/features/datasources/azuremonitor.md:
```diff
@@ -0,0 +1,271 @@
+- Microsoft.Network/networkInterfaces - Bytes sent
+- Microsoft.Storage/storageAccounts - Used Capacity
+
+{{< docs-imagebox img="/img/docs/v60/azuremonitor-service-query-editor.png" class="docs-image--no-shadow" caption="Azure Monitor Query Editor" >}}
```
Broken image, but I guess you already know that

marefr commented on docs/sources/features/datasources/azuremonitor.md:
```diff
@@ -0,0 +1,271 @@
+
+Grafana alerting is supported for the Azure Monitor service. This is not Azure Alerts support. Read more about how alerting in Grafana works [here]({{< relref "alerting/rules.md" >}}).
+
+{{< docs-imagebox img="/img/docs/v60/azuremonitor-alerting.png" class="docs-image--no-shadow" caption="Azure Monitor Alerting" >}}
```
Broken image, but I guess you already know that

marefr commented on docs/sources/features/datasources/azuremonitor.md:
```diff
@@ -0,0 +1,271 @@
+
+## Querying the Application Insights Service
+
+{{< docs-imagebox img="/img/docs/v60/appinsights-service-query-editor.png" class="docs-image--no-shadow" caption="Application Insights Query Editor" >}}
```
Broken image, but I guess you already know that

marefr commented on docs/sources/features/datasources/azuremonitor.md:
```diff
@@ -0,0 +1,271 @@
+- Passing in metric name variable: `AppInsightsGroupBys(requests/count)`
+- Chaining template variables: `AppInsightsGroupBys($metricnames)`
+
+{{< docs-imagebox img="/img/docs/v60/appinsights-service-variables.png" class="docs-image--no-shadow" caption="Nested Application Insights Template Variables" >}}
```
Broken image, but I guess you already know that

marefr commented on docs/sources/features/datasources/azuremonitor.md:
````diff
@@ -0,0 +1,271 @@
+| order by TimeGenerated desc
+```
+
+{{< docs-imagebox img="/img/docs/v60/azureloganalytics-service-query-editor.png" class="docs-image--no-shadow" caption="Azure Log Analytics Query Editor" >}}
````
Broken image, but I guess you already know that

marefr commented on docs/sources/features/datasources/azuremonitor.md:
```diff
@@ -0,0 +1,271 @@
+
+- `$__from` - Returns the From datetime from the Grafana picker. Example: `datetime(2018-06-05T18:09:58.907Z)`.
+- `$__to` - Returns the From datetime from the Grafana picker. Example: `datetime(2018-06-05T20:09:58.907Z)`.
+- `$__interval` - Grafana calculates the minimum time grain that can be used to group by time in queries. More details on how it works [here](http://docs.grafana.org/reference/templating/#the-interval-variable). It returns a time grain like `5m` or `1h` that can be used in the bin function. E.g. `summarize count() by bin(TimeGenerated, $__interval)`
```
Sort of broken link, wrong anchor? Should probable be relative link as well.

daniellee replied:
Was from the docs from the repo - missed making that one relative.

marefr reviewed (changes requested):
Sorry for the broken images comments. You'll probably already know about them :) Found some broken links that needs to be fixed. I would also want a link from What's new in v6.0 to Using Azure Monitor.

Besides that looks great :+1:

marefr committed:
docs: layout fixes

marefr committed:
docs: landing page update

link to what's new in v6.0
add azure monitor and loki to datasource list
change stackdriver icon to a bigger one

marefr commented:
For images and updated sass on docs landing page please include changes from branch: https://github.com/grafana/grafana.org/tree/docs_landing_page_v60

daniellee committed:
docs: tweaks to AzureMonitor docs

marefr committed:
docs: link to azure monitor from what's new in v6.0
